it seems odd to me that this would be the root cause since exports have been named like this for ages. i do like the name change regardless though. hope it fixes it!
```
======================================================================
ERROR: Ensure that we can export stock properties in a form export
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/commcare-hq-ro/corehq/apps/export/tests/test_get_export_file.py", line 257, in test_form_stock_columns
    with writer.open([export_instance]):
  File "/usr/local/lib/python2.7/contextlib.py", line 17, in __enter__
    return self.gen.next()
  File "/mnt/commcare-hq-ro/corehq/apps/export/export.py", line 65, in open
    name = safe_filename(name)
  File "/mnt/commcare-hq-ro/corehq/util/files.py", line 22, in safe_filename
    filename = filename if isinstance(filename, unicode) else filename.decode('utf8')
AttributeError: 'NoneType' object has no attribute 'decode'
```

--- a/corehq/apps/export/export.py
+++ b/corehq/apps/export/export.py
@@ -59,7 +59,7 @@ class _Writer(object):
         assert self._path is None
 
         if len(export_instances) == 1:
-            name = export_instances[0].name
+            name = export_instances[0].name or ''
         else:
             name = ''
         name = safe_filename(name)

